help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,141 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![Moengage Destination Authentication](../../assets/catalog/mobile-engagement/moengage/authentication.png)
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Moengage Destination Authentication](../../assets/catalog/mobile-engagement/moengage/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Account ID]**: Your app *data center*.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to streaming segment export destinations](../../ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from [!DNL Adobe Experience Platform] to the [!DNL Moengage] destination, you need to go through the field mapping step.
+
+Mapping consists of creating a link between your [!DNL Experience Data Model] (XDM) schema fields in your [!DNL Platform] account, and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL Braze] destination fields, follow these steps:
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Checkbox]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/segments.png)
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/mapping.png)
+
+In the [!UICONTROL Source Field] section, click the arrow button next to the empty field.
```

Comment:
minor, but please change click to select. this is because some users might not be using the classic mouse for work.

Reply:
done